tests/tests.py

```diff
@@ -500,16 +500,16 @@ def test_versatile_image_field_picklability(self):
             '/media/__sized__/python-logo-thumbnail-100x100-70.jpg'
         )
         pickled_state = self.jpg.image.__getstate__()
-        self.assertEqual(
-            pickled_state,
-            {
-                '_create_on_demand': False,
-                '_committed': True,
-                '_file': None,
-                'name': 'python-logo.jpg',
-                'closed': False
-            }
-        )
+        # self.assertEqual(
+        #     pickled_state,
+        #     {
+        #         '_create_on_demand': False,
+        #         '_committed': True,
+        #         '_file': None,
+        #         'name': 'python-logo.jpg',
+        #         'closed': False
+        #     }
+        # )
```

Comment:
```suggestion
        pickled_state.pop("field", None)
        pickled_state.pop("instance", None)
        pickled_state.pop("storage", None)
        compare(
            actual=pickled_state,
            expected={
                '_create_on_demand': False,
                '_committed': True,
                '_file': None,
                'name': 'python-logo.jpg',
                'closed': False,
                '_ppoi_value': (0.25, 0.25)
            }
        )
```

Note: for comparison of dictionary and complex data, I usually use the awesome `compare()` function of testfixtures python packages. The error traceback generated by testfixtures' `compare` is much more useful and easier to understand than the standard version of unittest2 and pytest

https://testfixtures.readthedocs.io/en/latest/comparing.html